src/components/DragDrop/DragDrop.tsx

```diff
@@ -78,60 +77,63 @@ export const DragDrop = ({
   /**
    * Target drag drop and drag drop inner DOM elements
    */
-  const dragDropInnerRef = useRef() as MutableRefObject<HTMLDivElement>;
-  const dragDropRef = useRef() as MutableRefObject<HTMLElement>;
+  const dragDropInnerRef = useRef<HTMLDivElement>(null);
+  const dragDropRef = useRef<HTMLElement | null>(null);
   /**
    * Set right and left gradients on tables
    */
   const setShadows = () => {
-    /**
-     * Target the actual drag drop inside drag drop.
-     */
-    const overflowListItems = Array.from(dragDropInnerRef.current.children);
-    let childrenWidth = 0;
-
-    overflowListItems.forEach(function (item) {
-      const itemMarginLeft = parseInt(
-        getComputedStyle(item).marginLeft.slice(0, -2),
-      );
-      childrenWidth += item.clientWidth + itemMarginLeft;
-    });
-    if (overflowListItems) {
+    if (dragDropInnerRef.current) {
```

Comment:
Now that we're not type casting (the `as` has been removed), we have to handle the `null` case. Handling it here by adding a top-level `if`.